### ISSUE TITLE
RuntimeError: Version file not found: xgrammar/python/xgrammar/version.py

### ISSUE THREAD

**Issue by lifeiteng** [mlc-ai/xgrammar#105]
```
mlc-ai/xgrammar/python git:(main) python3 -m pip install .
Processing /home/lifeiteng/code/LLM/mlc-ai/xgrammar/python
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error
  
  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [11 lines of output]
      Traceback (most recent call last):
        File "<string>", line 2, in <module>
        File "<pip-setuptools-caller>", line 34, in <module>
        File "/home/lifeiteng/code/LLM/mlc-ai/xgrammar/python/setup.py", line 113, in <module>
          main()
        File "/home/lifeiteng/code/LLM/mlc-ai/xgrammar/python/setup.py", line 90, in main
          version=get_version(),
                  ^^^^^^^^^^^^^
        File "/home/lifeiteng/code/LLM/mlc-ai/xgrammar/python/setup.py", line 17, in get_version
          raise RuntimeError(msg)
      RuntimeError: Version file not found: /home/lifeiteng/code/LLM/mlc-ai/xgrammar/python/xgrammar/version.py
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.
```

**Comment by zanderjiang** [comment]
Fixed refer to #107

**Comment by Ubospica** [comment]
It is fixed in #109
